src/librustdoc/html/static/main.js

```diff
@@ -445,6 +488,20 @@ if (!DOMTokenList.prototype.remove) {
         var OUTPUT_DATA = 1;
         var params = getQueryStringParams();
 
+        // Loading the crate filter if any.
+        var savedCrates = getCurrentValue("rustdoc-saved-filter-crate");
+        if (savedCrates !== null) {
+            var elems = getSavedCrates(savedCrates);
+            if (elems.hasOwnProperty(getCratesHash())) {
+                onEach(window.crates, function(e) {
+                    if (e === elems[window.hashCode]) {
```

Comment:
Can we avoid storing the crates hash in a global variable and simply have it in a variable here? It looks like this function should presumably be called only once so presumably that's equivalent performance wise?

Reply:
Sure.